pkg/api/v1alpha1/hostosconfig.go

```diff
@@ -0,0 +1,49 @@
+package v1alpha1
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+)
+
+func validateHostOSConfig(config *HostOSConfiguration, machineName string) error {
+	if config == nil {
+		return nil
+	}
+
+	if err := validateVSphereMachineConfigNTPServers(config, machineName); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func validateVSphereMachineConfigNTPServers(config *HostOSConfiguration, machineName string) error {
```

Comment:
Instead of passing the machine name just for errors can we have the caller wrap it with some context? It might also be useful to, at the caller site, specify the type of object.

Reply:
fixed